secant/Models/FeatureFlags.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  FeatureFlags.swift
+//  secant
+//
+//  Created by Michal Fousek on 23.02.2023.
+//
+
+enum FeatureFlag: String, CaseIterable, Codable {
```

Comment:
```suggestion
enum Setting: String, CaseIterable, Codable {
```

Reply:
This enum has one specific purpose. And that is to hold list of feature flags. In future we may have some other things (AB tests, server side UI, whatever....). And data structures representing these things will probably have specific purposes and names. For example:
```
enum ABTestName: String {
....
}
```

Then all the specific things will be inside `WalletConfig` structure. `Settings` seems like alternative name for `WalletConfig`. But it seems to generic for enum that holds list of feature flags.